components/__tests__/__snapshots__/locale-chooser.js.snap

```diff
@@ -334,21 +329,25 @@ exports[`opened state 1`] = `
             </glamorous(li)>
             <glamorous(li)>
               <li
-                className="css-l0f4u7"
+                className="css-1uz022"
               >
                 <glamorous(a)
-                  href="about://es.blank"
+                  aria-label="Change localization to Español"
+                  href="about://es.undefinedblank"
                 >
                   <a
-                    className="css-m4ei6j"
-                    href="about://es.blank"
+                    aria-label="Change localization to Español"
+                    className="css-1c1bzyg"
+                    href="about://es.undefinedblank"
                   >
                     <div>
                       <EsSvg
+                        alt="Español"
```

Comment:
It would be beneficial to add `lang="es"` where you use languages that don't match the `html[lang]` attribute so the screen reader knows what language to interpret it as.